packages/portfolio-contract/src/resolver/resolver.exo.ts

```diff
@@ -207,6 +209,7 @@ export const prepareResolverKit = (
         completePendingTransaction(
           txId: TxId,
           status: Exclude<TxStatus, 'pending'> = TxStatus.SUCCESS,
+          rejectionReason?: string,
```

Comment:
wow... I assumed the `rejectionReason` was already published if it was provided.